homeassistant/components/transmission/const.py

```diff
@@ -2,14 +2,14 @@
 DOMAIN = "transmission"
 
 SENSOR_TYPES = {
-    "active_torrents": ["Active Torrents", None],
+    "active_torrents": ["Active Torrents", "Counts"],
```

Comment:
`Counts` seems like an awkward unit. What about `torrents`?

Reply:
Changed to Torrents.